src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -264,7 +270,7 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
                     <span className="private-chat-icon">
                       {numberOfUnreadMessages > 0 && (
                         <div className="notification-card">
-                          {numberOfUnreadMessages}
+                          {numberOfUnreadMessages} {playAudio}
```

Comment:
This isn't required here, as the `useEffect` described above will run regardless

```suggestion
                          {numberOfUnreadMessages}
```